github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -125,10 +126,70 @@ periodics:
       - "/usr/local/bin/runner.sh"
       - "/bin/bash"
       - "-c"
-      - >
+      - |
+        cat $QUAY_PASSWORD | podman login --username $(<$QUAY_USER) --password-stdin quay.io &&
+        ./hack/bump-centos-version.sh &&
+        GIT_ASKPASS=../project-infra/hack/git-askpass.sh ../project-infra/hack/git-pr.sh -c "PHASES=linux BYPASS_PMAN_CHANGE_CHECK=true ./publish.sh" -r kubevirtci -b bump-centos-stream -T main -p $(pwd) -s "Automatic bump of CentOS Stream to latest" &&
+        # For passing centos image tag to dependent (s390x) prow job
+        SHORT_SHA=$(git rev-parse --short HEAD) &&
+        image_tag=$(cat cluster-provision/k8s/base-image | cut -d ':' -f 2) &&
+        echo "$image_tag" > amd64-centos9-$SHORT_SHA &&
+        gsutil cp ./amd64-centos9-$SHORT_SHA gs://kubevirt-prow/release/kubevirt/kubevirtci/amd64-centos9-$SHORT_SHA
+      # docker-in-docker needs privileged mode
+      env:
+      - name: GIMME_GO_VERSION
+        value: "1.22.5"
+      securityContext:
+        privileged: true
+      resources:
+        requests:
+          memory: "29Gi"
+- name: periodic-kubevirtci-bump-centos-base-s390x
+  cron: "0 5 * * 2"
+  annotations:
+    testgrid-create-test-group: "false"
+  decorate: true
+  decoration_config:
+    timeout: 2h
+  max_concurrency: 1
+  extra_refs:
+  - org: kubevirt
+    repo: kubevirtci
+    base_ref: main
+    workdir: true
+  labels:
+    preset-podman-in-container-enabled: "true"
+    preset-kubevirtci-quay-credential: "true"
+    preset-gcs-credentials: "true"
+  cluster: prow-s390x-workloads
+  spec:
+    containers:
+    - image: quay.io/kubevirtci/golang:v20241014-80f340c
+      command:
+      - "/usr/local/bin/runner.sh"
+      - "/bin/bash"
+      - "-c"
+      - |
+        # For getting centos image tag from amd64 prow job and use same for s390x and manifest-list images
+        SHORT_SHA=$(git rev-parse --short HEAD) &&
+        GCS_FILE_PATH=release/kubevirt/kubevirtci/amd64-centos9-$SHORT_SHA &&
```

Comment:
Why don't we just curl for the expected public path? then we could avoid the need for gcs credentials etc. here?

Like how we do here in the publish - https://github.com/kubevirt/kubevirtci/blob/1048544d270ecef75b4972bce89d8dd97434e883/publish.sh#L9

Reply:
You are right for this part. I've changed the code locally, but later realized that as we anyway need credentials as we are removing the file `        rm_gcs_file kubevirt-prow "$GCS_FILE_PATH"` below, probably we can leave this as it is.